BirdQuiz/QuizFrontend.py

```diff
@@ -172,7 +174,14 @@ def end(self):
         self.root.mainloop()
 
     
-
+    def add_to_listbox(self, inputbox, listbox):
+        """Moves text from the inputbox to the listbox with some text validation"""
+        text = inputbox.get("1.0", "end").strip()
```

Comment:
I like the auto-capitalization, so can you add `capwords` like they recommend in [this post](https://stackoverflow.com/questions/8199966/python-title-with-apostrophes)?

Reply:
to be fixed in next PR